src/Stdraum.cpp

```diff
@@ -70,7 +70,7 @@ XY BeraterSprechblasenOffset [] =
      XY(82, 385)          // 15 = Player 4
    };
 
-//F�r den Taschenrechner
+//Für den Taschenrechner
```

Comment:
oops I didn't intend to do that. It fixes the file from corrupted ascii to proper UTF8